### PR TITLE
[mongodb] type cursor method & support flow v132+

### PR DESCRIPTION
**Pull request by moroine**
- Links to documentation: http://mongodb.github.io/node-mongodb-native/3.5/api/MongoClient.html#.connect
- Link to GitHub or NPM:
- Type of contribution: addition
- Other notes:

### CODE REVIEW

**Comment by moroine** [comment]
@pascalduez could you have a look at this?

File: definitions/npm/mongodb_v3.5.x/flow_v0.114.x-/mongodb_v3.5.x.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
 // @flow
 
 declare module 'mongodb' {
+  import type { Readable } from 'stream';
```
Comment (AndrewSouthpaw) [inline definitions/npm/mongodb_v3.5.x/flow_v0.114.x-/mongodb_v3.5.x.js:4]: This will import as `any`, see the `CONTRIBUTING.md` for more details.

**Review by AndrewSouthpaw — CHANGES_REQUESTED** [review]
Looks like a step in the right direction! Thanks for that work. One small thing that you'll need to fix...

File: definitions/npm/mongodb_v3.5.x/flow_v0.114.x-/mongodb_v3.5.x.js (hunk 1 above)
Comment (pascalduez) [reply definitions/npm/mongodb_v3.5.x/flow_v0.114.x-/mongodb_v3.5.x.js:4]: Event with node built ins like this? I have a doubt... :thinking:

Comment (moroine) [reply definitions/npm/mongodb_v3.5.x/flow_v0.114.x-/mongodb_v3.5.x.js:4]: I've added some tests (I was lazy yesterday). From what I tested it's working properly with node built ins but it should be under `declare module` statement

**Review by pascalduez — APPROVED** [review]

File: definitions/npm/mongodb_v3.5.x/flow_v0.114.x-/mongodb_v3.5.x.js (hunk 1 above)
Comment (AndrewSouthpaw) [reply definitions/npm/mongodb_v3.5.x/flow_v0.114.x-/mongodb_v3.5.x.js:4]: Ah! Sorry, I wasn't paying attention that it's a node core package.

**Review by AndrewSouthpaw — APPROVED** [review]

**Pull request merged**